docs/commands.md

```diff
@@ -105,11 +105,15 @@ cpl copy-image-from-upstream -a $APP_NAME --upstream-token $UPSTREAM_TOKEN --ima
 
```

Comment:
All modifications in this file come from the command files, so it might be best to always ignore this and just go straight to the source (this file is always automatically generated based on the source).